src/admin/index.js

```diff
@@ -44,6 +44,16 @@ const findTopicPartitions = async (cluster, topic) => {
     .sort()
 }
 
+/**
+ *
+ * @param {Object} params
+ * @param {import("../../types").Logger=} params.logger
```

Comment:
This trailing `=` seems to make the parameter optional, but then later it is assumed it is defined:

```
const logger = rootLogger.namespace('Admin')
```

(On the nit-picky side: I find the `@param {type} [optionalParameter]` syntax easier to understand and more visible than a trailing `=` ...)

Reply:
> This trailing = seems to make the parameter optional, but then later it is assumed it is defined:

Whoops! For some reason my brain read `logger: rootLogger` as `logger` having a default value. 🙈

> (On the nit-picky side: I find the @param {type} [optionalParameter] syntax easier to understand and more visible than a trailing = ...)

I think I agree. I'm gonna refactor to use that syntax instead. I think I adopted the trailing equals because I wanted to add the default value, but then I decided not to because vscode doesn't use the default values anyway and it's so much duplication.

Reply:
> I think I adopted the trailing equals because I wanted to add the default value, but then I decided not to because vscode doesn't use the default values anyway and it's so much duplication.

Like this: `@param {Foo} [optionalParameter=defaultValue]`?

Reply:
Fixed the syntax in `f6e215f` and the issue with the logger not being required in `048cb66`.